src/examples/push_existing_assets.yml

```diff
@@ -0,0 +1,24 @@
+description: Deploy with artifacts created in previous CircleCI jobs
+
+usage:
```

Comment:
Example appears incomplete. It appears there should maybe be a step in the build job to persist to the workspace used by the CF job. Would also recommend re-ordering the example so that version is at the top, followed by the imported orbs. some of the other examples have the same ordering issue.